index.html

```diff
@@ -91,43 +95,65 @@ <h2>
         Badge model
       </h2>
       <p>
-        There is a single global badge associated with each <a data-cite=
-        "appmanifest#installable-web-applications">web application</a>. At any time, the application's badge
-        is set to one of the following values:
+        A <dfn>badge</dfn> may be one of the following values:
       </p>
       <ul>
-        <li>The special value <a>nothing</a>.
+        <li>The special value <dfn>nothing</dfn>.
         </li>
-        <li>The special value <a>flag</a>.
+        <li>The special value <dfn>flag</dfn>.
         </li>
         <li>An <code><a data-cite="WEBIDL#idl-unsigned-long-long">unsigned long
         long</a></code> value, which MUST NOT be 0.
         </li>
       </ul>
       <p>
-        The user agent MUST initialize each application's badge to
-        <a>nothing</a>. The user agent MAY reset an application's badge to
-        <a>nothing</a> at its discretion (for example, when the system is
-        restarted).
+        Each <a data-cite="dom#concept-document">document</a> and each
+        <a data-cite="appmanifest#installable-web-applications">web
+        application</a> is associated with a <a>badge</a> value, which is
+        initialized to <a>nothing</a>.
+      </p>
+      <p>
+        The user agent MAY reset an application's badge to <a>nothing</a> at
+        its discretion (for example, when the system is restarted).
       </p>
       <p>
-        If the application's badge is <dfn>nothing</dfn>, the badge is said to be
+        If a <a>badge</a> is <a>nothing</a>, it is said to be
         "<dfn>cleared</dfn>". Otherwise, it is said to be "<dfn>set</dfn>".
       </p>
-      <p>Note: The API is set only, so the badge data cannot be used to identify users.</p>
     </section>
     <section>
       <h2>
         Badge display
       </h2>
       <p>
-        When the application's flag is <a>set</a>, the user agent SHOULD display
-        the application's badge alongside the symbolic representation of the
-        application in the user's operating system (for example, as a small
-        overlay on top of the application's icon).
+        When a document's badge is <a>set</a>, if the <a data-cite=
```

Comment:
Remove data-cite, xref will auto link it to html for you. Same with the rest.

Reply:
Ok, we need to figure out why it wasn't exported. That's a bug in one of the two specs.